javacord-core/src/main/java/org/javacord/core/util/gateway/DiscordWebSocketAdapter.java

```diff
@@ -703,6 +704,10 @@ private void sendIdentify(WebSocket websocket) {
                 .put("$device", "Javacord")
                 .put("$referrer", "")
                 .put("$referring_domain", "");
+
+        data.put("intents",api.getIntent().isEmpty()
+                ? 32509 : Intent.calculateBitmask(api.getIntent().toArray(new Intent[0])));
```

Comment:
* What is `32509`? If that's the bitmask for all non-privileged intents: See my suggestion in `DiscordApiBuilderDelegateImpl`.
* If you default `api.getIntent().isEmpty()` to `32509`, it is impossible for the user to specify "I don't want any intents". Just set whatever the result of `api.getIntents()` is.
```suggestion
        data.put("intents", Intent.calculateBitmask(api.getIntent().toArray(new Intent[0])));
```